.php_cs.dist

```diff
@@ -9,11 +18,18 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'array_syntax' => array('syntax' => 'long'),
+        'braces' => array('allow_single_line_closure' => true),
+        'heredoc_to_nowdoc' => false,
+        'no_break_comment' => false,
+        'no_unreachable_default_argument_value' => false,
+        'phpdoc_annotation_without_dot' => false,
```

Comment:
Why? We've just fixed the codebase for that.

Reply:
Ah I prepared this by checking this file against the version on master,
but did it before https://github.com/symfony/symfony/commit/7a97b49436b20c9664f3e286f8c174bc1aae0364#diff-8b6c0c683d533f32b31ce28cee1bd3e6 was merged and overlooked it later on.
Will revert!